src/components/ChainSelectionPopover/ChainSelectionPopover.styles.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components/macro";
+
+import breakPoints from "../../style/breakpoints";
+import { BorderlessButtonStyle } from "../../style/mixins";
+
+/**
+ * @remarks props are used to manipulate the positioning of components
+ */
+type ContainerProps = {
+  open: boolean;
+  shiftLeft: boolean;
+};
+
+export const Container = styled.div<ContainerProps>`
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  width: 16rem;
+  height: 14rem;
+  top: 5rem;
+  right: 3.75rem;
+  padding-top: 0.5rem;
+  transform: ${(props) => (props.shiftLeft ? "translate(-16.75rem, 0)" : "0")};
+  color: ${(props) => props.theme.colors.darkSubText};
+  background-color: ${({ theme }) =>
+    theme.name === "dark" ? theme.colors.darkGrey : theme.colors.black};
+  border: ${(props) => props.theme.colors.borderGrey} 1px solid;
+  border-radius: 0.5rem;
+  z-index: 1000;
+
+  @media ${breakPoints.tabletPortraitUp} {
+    right: 12rem;
+  }
+
+  @media ${breakPoints.tabletPortraitUpMax} {
+    right: 10rem;
+  }
+
+  @media ${breakPoints.phoneOnly} {
+    top: 4rem;
+    right: 4rem;
+  }
+`;
+
+type ChainSelectionContainerType = {
+  $overflow: boolean;
+};
+
+export const NetworksContainer = styled.div<ChainSelectionContainerType>`
+  margin: 0.5rem 0 0 0;
+  display: flex;
+  flex-direction: column;
+  height: 2.5rem;
+`;
+
+type ButtonStyleProps = {
+  $isActive: boolean;
+};
+
+export const NetworkButton = styled.button<ButtonStyleProps>`
```

Comment:
You're missing background when active. Check design:

![Screenshot 2023-06-03 at 13 24 00](https://github.com/airswap/airswap-web/assets/86911296/233ea4aa-1dc0-48f2-aa05-3d79c78d9cc5)

Reply:
Are you sure? On my end it looks like the background does change when active on both light and dark mode

Reply:
This error `MetaMask - RPC Error: Expected 0x-prefixed, unpadded, non-zero hexadecimal string 'chainId'. Received` happens because in type `ChainParams` in chainParams.ts, `chainId` is set to `number`. If it's set to string, this error won't happen, and MetaMask will prompt the user to add the chain.

I'm just going to remove this feature since we have a disagreement about the type of `chainId`

Reply:
Oh yes I was on goerli.